model/task/task.go

```diff
@@ -1001,6 +1001,9 @@ func (t *Task) String() (taskStruct string) {
 
 // Insert writes the b to the db.
 func (t *Task) Insert() error {
+	if t.CreateTime == util.ZeroTime {
+		grip.Warning(message.WrapStack(1, fmt.Sprintf("task %s inserted with zero creation time", t.Id)))
```

Comment:
I think there are helpers that will allow you to elide both the conditional (e.g. use a when log,) and also the sprintf

Reply:
fixed